ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -55,6 +55,21 @@ Delete Data Science Project
     Handle Deletion Confirmation Modal    item_title=${project_title}    item_type=project
     ...                                   press_cancel=${press_cancel}
     ...   additional_msg=It will destroy all workbenches, storages, data connections and other resources in ${project_title}
+    Wait Until Data Science Project Is Deleted    project_title=${project_title}
```

Comment:
since you moved this kw here, could you pls remove it from the test case`Verify User Can Delete A Data Science Project`?

Reply:
Done